### ISSUE TITLE
[OTHER] Social Preview Card

### ISSUE THREAD

**Issue by narayan954** [narayan954/dummygram#232]
### What would you like to share?

Github allows for our repository to have a social media preview and this could've a significant effect in spreading our application socially. We can use their template and our logo to create an appealing social media preview.

### Additional information

![social preview](https://user-images.githubusercontent.com/77617189/222884514-a403d798-9b4e-41a1-8025-129f65b11926.png)
![repository-open-graph-template](https://user-images.githubusercontent.com/77617189/222884582-78104668-f54b-4dee-924d-8124a6a03622.png)

**Comment by narayan954** [comment]
https://www.freecodecamp.org/news/how-to-add-a-social-media-image-to-your-github-project/

**Comment by arjunchoudhury07** [comment]
i want to contribute under JWOC

**Comment by arjunchoudhury07** [comment]
I want to contribute under JWOC

**Comment by narayan954** [comment]
> I want to contribute under JWOC

go ahead! :)

**Comment by arjunchoudhury07** [comment]
![Dummygram](https://user-images.githubusercontent.com/92199877/224552735-fc75cd92-72a7-4544-92a6-f8e3cb017edc.png)
![Dummygram (1)](https://user-images.githubusercontent.com/92199877/224552948-dc22fe0a-4937-4172-b549-1c43bbf88d93.png)

**Comment by arjunchoudhury07** [comment]
I have made the images but only you can change the settings. I cannot make any PRs

**Comment by narayan954** [comment]
> I have made the images but only you can change the settings. I cannot make any PRs

yeah right! You can add this to assets folder in src.... and make a pr for that! (first one looks nice right?)
Also, keep in mind the borders are cropped out in preview so you may want to adjust the picture according to that.
example image: ![github-social-media-image](https://user-images.githubusercontent.com/77617189/224566600-bc667889-4ef6-4919-924c-1597558be709.jpg)
